Source/AssetRipper.IO.Files/Utils/FileUtils.cs

```diff
@@ -39,10 +39,10 @@ public static string GetUniqueName(string dirPath, string fileName, int maxNameL
 			string? name = null;
 			int maxLength = maxNameLength - 4;
```

Comment:
This line doesn't make sense to me. I can't see why I (or mafaca) wrote it this way because `.meta` is 5 bytes. I think it can be removed.